### PR TITLE
Feature/bau 548 decimal points

### PR DESCRIPTION
**Pull request by shakes90**

### CODE REVIEW

File: src/GovUk.Education.ExploreEducationStatistics.Data.Importer/Services/ImporterMetaService.cs [hunk 1]
```diff
@@ -76,7 +77,8 @@ public static MetaRow GetMetaRow(List<string> cols, DataRow row)
                 FilterGroupingColumn = values[3],
                 FilterHint = values[4],
                 IndicatorGrouping = values[5],
-                IndicatorUnit = EnumUtil.GetFromString<Unit>(values[6] ?? "")
+                IndicatorUnit = EnumUtil.GetFromString<Unit>(values[6] ?? ""),
+                DecimalPlaces = values[7] == null ? (int?) null : int.Parse(values[7])
```
Comment (ntsim) [inline src/GovUk.Education.ExploreEducationStatistics.Data.Importer/Services/ImporterMetaService.cs:81]: Do we actually do any meta file validation to ensure it's in the correct format? I'm guessing we don't report any problems back to the user if there is an issue?

File: src/explore-education-statistics-common/src/modules/charts/components/MapBlockInternal.tsx [hunk 2]
```diff
@@ -445,12 +445,12 @@ export const MapBlockInternal = ({
         if (feature.properties) {
           const content = Object.entries(feature.properties.measures)
             .map(([id, value]) => ({
-              ...(labels[id] || { label: '', unit: '' }),
+              ...(labels[id] || { label: '', unit: '', decimalPlaces: 0 }),
```
Comment (ntsim) [inline src/explore-education-statistics-common/src/modules/charts/components/MapBlockInternal.tsx:448]: Should we be defaulting to 0 here?

File: src/GovUk.Education.ExploreEducationStatistics.Data.Importer/Services/ImporterMetaService.cs (hunk 1 above)
Comment (shakes90) [reply src/GovUk.Education.ExploreEducationStatistics.Data.Importer/Services/ImporterMetaService.cs:81]: If it's non-numeric then it'll be caught by the upload validation but no val on max or min values.

File: src/explore-education-statistics-common/src/modules/charts/components/MapBlockInternal.tsx (hunk 2 above)
Comment (shakes90) [reply src/explore-education-statistics-common/src/modules/charts/components/MapBlockInternal.tsx]: Changed to 2

**Pull request merged**